blosc2/CMakeLists.txt

```diff
@@ -18,13 +25,9 @@ else()
     set(BLOSC_INSTALL ON)
```

Comment:
Do you want to install the `blosc` files alongside the python files, particularly since you link to the static library. If so in the `pyproject.toml` change `tool.scikit-build.wheel.install-dir` to `blosc2`, and the `install(blosc2_ext)` to `.` accordingly.